presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1265,17 +1265,17 @@ public void testShowColumnsFromPartitions()
         assertQueryFails(
                 getSession(),
                 "SHOW COLUMNS FROM \"$partitions\"",
-                ".*Table 'hive.tpch.\\$partitions' does not exist");
+                ".*Table '.*.tpch\\.\\$partitions' does not exist");
```

Comment:
Why don't you escape the first '.' character, but only the rest: `.*\\.tpch\\.`
You escape it on L1273 below, but not in all changes.

Reply:
The first is not going to be escaped but use `.*` to indicate any match. the rest one is want to use real `.` literal match

Reply:
Actually, not the first '.', but the second '.' character. I understand that you are doing literal '.' matches later, but here you use `.*.tpch\\.`, I am asking why not `.*\\.tpch\\.`? On L1273 you use the latter, so there is some inconsistency.

Reply:
@nezihyigitbasi  Good catch!